README.md

```diff
@@ -15,7 +15,7 @@ Install and run:
     * `python3 -m venv venv`
     * `source venv/bin/activate`
 * Run supporting infrastructure using Docker
-    * `./setup-docker.sh` 
-    * `docker-compose up -d`
+    * See the instructions in the [killrvideo-docker-common][https://github.com/KillrVideo/killrvideo-docker-common] repository for running the supporting infrastructure 
```

Comment:
Link is not rendered by github. I think the markdown is not correct here.
![image](https://user-images.githubusercontent.com/1742301/63095771-5aaa6380-bf6c-11e9-9063-891db454f50e.png)